air-quality-ui/package.json

```diff
@@ -14,6 +14,9 @@
     "test:ci": "jest --ci --json --coverage --testLocationInResults --outputFile=report.json"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
```

Comment:
maybe I missed it. but where is the emotion packages being used?

Reply:
It's required by the Mui date picker package, throws an error if they're not there so I think they're used by it internally